sql-plugin/src/main/scala/com/nvidia/spark/rapids/parquet/GpuParquetScan.scala

```diff
@@ -2198,17 +2204,18 @@ class MultiFileParquetPartitionReader(
   implicit def ParquetSingleDataBlockMeta(in: ExtraInfo): ParquetExtraInfo =
     in.asInstanceOf[ParquetExtraInfo]
 
-  // The runner to copy blocks to offset of HostMemoryBuffer
+  // The runner to copy blocks to offset of HostMemoryBuffer.
```

Comment:
what is our plan to fix this? We use this type of runner, so I'd like it to use the resource acquisition system.